dashboards/nextcloud/nextcloud.json

```diff
@@ -0,0 +1,483 @@
+			  "id": "viz.markdown"
+			},
+			"rawConfiguration": {
+			  "text": "![logo](https://symbols.getvecta.com/stencil_25/59_nextcloud.957608d8aa.png)"
+			}
+		  },
+		  {
+			"title": "Day to Day Active Users",
+			"layout": {
+			  "column": 3,
+			  "row": 1,
+			  "width": 6,
+			  "height": 3
+			},
+			"linkedEntityGuids": null,
+			"visualization": {
+			  "id": "viz.line"
+			},
+			"rawConfiguration": {
+			  "facet": {
+				"showOtherSeries": false
+			  },
+			  "legend": {
+				"enabled": true
+			  },
+			  "nrqlQueries": [
+				{
+				  "accountId": 0,
+				  "query": "FROM Metric SELECT latest(nextcloud_active_users_daily_total) FACET dateOf(timestamp) TIMESERIES AUTO"
+				}
+			  ],
+			  "platformOptions": {
+				"ignoreTimeRange": false
+			  },
+			  "yAxisLeft": {
+				"zero": true
+			  }
+			}
+		  },
+		  {
+			"title": "User Accounts",
+			"layout": {
+			  "column": 9,
+			  "row": 1,
+			  "width": 2,
+			  "height": 5
+			},
+			"linkedEntityGuids": null,
+			"visualization": {
+			  "id": "viz.billboard"
+			},
+			"rawConfiguration": {
+			  "facet": {
+				"showOtherSeries": false
+			  },
+			  "nrqlQueries": [
+				{
+				  "accountId": 0,
+				  "query": "FROM Metric SELECT latest(nextcloud_users_total) As 'Total Users', latest(nextcloud_active_users_total) As 'Total Active Users', latest(nextcloud_active_users_daily_total) As 'Active Users Daily', latest(nextcloud_active_users_hourly_total) As 'Active Users Hourly'"
+				}
+			  ],
+			  "platformOptions": {
+				"ignoreTimeRange": false
+			  }
+			}
+		  },
+		  {
+			"title": "Federation Shares",
+			"layout": {
+			  "column": 11,
+			  "row": 1,
+			  "width": 2,
+			  "height": 3
+			},
+			"linkedEntityGuids": null,
+			"visualization": {
+			  "id": "viz.billboard"
+			},
+			"rawConfiguration": {
+			  "facet": {
+				"showOtherSeries": false
+			  },
+			  "nrqlQueries": [
+				{
+				  "accountId": 0,
+				  "query": "FROM Metric SELECT latest(nextcloud_shares_federated_total) As 'Shares Federated' facet direction"
+				}
+			  ],
+			  "platformOptions": {
+				"ignoreTimeRange": false
+			  }
+			}
+		  },
+		  {
+			"title": "",
+			"layout": {
+			  "column": 1,
+			  "row": 3,
+			  "width": 2,
+			  "height": 3
+			},
+			"linkedEntityGuids": null,
+			"visualization": {
+			  "id": "viz.markdown"
+			},
+			"rawConfiguration": {
+			  "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/catalog-pack-details/nr1-core/home?state=6538d574-b26d-0e36-ae65-f402e7b8e8f2).\n\nUnable to find data in your dashboard? - [Troubleshoot here](\nhttps://docs.newrelic.com/docs/infrastructure/prometheus-integrations/troubleshooting/no-data-appears-prometheus-integration/)\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=Nextcloud) here and let us know how we can improve it for you.\n"
```

Comment:
please update "Add Data" link to "https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration/"

Reply:
Updated.